### PR TITLE
Many fix, much wow

### PR DESCRIPTION
**Pull request by sugarPhlox**
- Add favicons.
- Added about section information.
- Fixed issue with Windows Phone and navigation bar.
- Linked to composer website in guide, and to daft labs website on about page.
- Graphical changes (new fonts, more simple front page).

http://screencast.com/t/uEx3xBOJXH

http://screencast.com/t/Y8rKRChHAx

### CODE REVIEW

File: views/about.lavender [hunk 1]
```diff
@@ -5,12 +5,30 @@ head
...
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-96x96.png", sizes="96x96")
+  link(rel="icon", type="image/png", href="/img/favicons/android-chrome-192x192.png", sizes="192x192")
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-16x16.png", sizes="16x16")
+  link(rel="manifest", href="/img/favicons/manifest.json")
+  meta(name="msapplication-TileColor", content="#cdb4cc")
+  meta(name="msapplication-TileImage", content="/img/favicons/mstile-144x144.png")
+  meta(name="theme-color", content="#796878")
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Oswald:400,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Lato:300,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Special+Elite')
```
Comment (mowchan) [inline views/about.lavender:28]: You can combine the google fonts stylesheets using a pipe.

http://fonts.googleapis.com/css?family=Oswald:400,700|Lato:300,700|Special+Elite

File: views/guide.lavender [hunk 2]
```diff
@@ -5,12 +5,30 @@ head
...
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-96x96.png", sizes="96x96")
+  link(rel="icon", type="image/png", href="/img/favicons/android-chrome-192x192.png", sizes="192x192")
+  link(rel="icon", type="image/png", href="/img/favicons/favicon-16x16.png", sizes="16x16")
+  link(rel="manifest", href="/img/favicons/manifest.json")
+  meta(name="msapplication-TileColor", content="#cdb4cc")
+  meta(name="msapplication-TileImage", content="/img/favicons/mstile-144x144.png")
+  meta(name="theme-color", content="#796878")
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Oswald:400,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Lato:300,700')
+  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Special+Elite')
```
Comment (mowchan) [inline views/guide.lavender:28]: Same

**Comment by mowchan** [comment]
Looks good. Would like to see some screenshots. :cake:

File: views/about.lavender [hunk 3]
```diff
@@ -46,7 +64,7 @@ body
               li.divider
               li
                 a(href='#') Language Reference
-          li.active
+          li.disabled
```
Comment (zlincoln) [inline views/about.lavender:67]: Why disable the about page?

Comment (zlincoln) [inline views/about.lavender:67]: Oh, I see - you're on the about page

**Comment by zlincoln** [comment]
:cake:

**Pull request merged**